Description: Unguarded variable
Author: Roland Clobus <rclobus@rclobus.nl>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=537718
 The variable child_stderr was not always filled, so extra guards for the NULL
 pointer were required.
--- a/libgksu/libgksu.c
+++ b/libgksu/libgksu.c
@@ -2951,7 +2951,8 @@
 
       if (WEXITSTATUS(status))
 	{
-          if (g_str_has_prefix(child_stderr, "Sorry, user "))
+          if (child_stderr != NULL && 
+              g_str_has_prefix(child_stderr, "Sorry, user "))
             {
               g_set_error (error, gksu_quark, GKSU_ERROR_NOT_ALLOWED,
                            _("The underlying authorization mechanism (sudo) "
